localstripe/resources.py

```diff
@@ -939,6 +939,29 @@ def _api_list_tax_ids(cls, id, **kwargs):
         obj = cls._api_retrieve(id)
         return obj.tax_ids
 
+    @classmethod
+    def _api_retrieve_tax_id(cls, id, tax_id, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        obj = cls._api_retrieve(id)
+        for txi in obj.tax_ids._list:
+            if txi.id == tax_id:
+                return txi
+        raise UserError(404, 'Customer ' + id + ' does not have a TaxId with '
+                             'ID ' + tax_id)
```

Comment:
Usually in these error message we use the English name of objects: "does not have a subscription", "no such invoice"... So I would use "does not have a tax ID" (even if it repeats the word ID), what do you think?


Stripe docs:
![stripe API docs](https://github.com/adrienverge/localstripe/assets/5244945/008a0d74-6ab3-429f-88b8-0422573656e9)

Reply:
ok, replaced `does not have a TaxId...` with `does not have a tax ID...`